README.md

```diff
@@ -1,46 +1,123 @@
-# etc-client
+# mantis - Scala client for Ethereum Classic
 
+*Mantis* is the name given to the new Ethereum Classic client produced by the Grothendieck Team.
+ 
+### Status - Beta 1 
+
+Continuous Integration Build Status [![CircleCI](https://circleci.com/gh/input-output-hk/etc-client/tree/master.svg?style=svg)](https://circleci.com/gh/input-output-hk/etc-client/tree/master)
+
+Unit Test Code Coverage Status [![Coverage Status](https://coveralls.io/repos/github/input-output-hk/etc-client/badge.svg?branch=master)](https://coveralls.io/github/input-output-hk/etc-client?branch=master)
+
+This version of the code supports 
 
-[![CircleCI](https://circleci.com/gh/input-output-hk/etc-client/tree/master.svg?style=svg)](https://circleci.com/gh/input-output-hk/etc-client/tree/master)
+  - fast sync (download a recent state trie snapshot and all blocks, this is the default behaviour)
+  - regular sync (download and execute every transaction in every block in the chain, this can be very slow)
+  - bootstrap sync (download a database for *mantis* preloaded with a recent version of the block chain, recommended for testing) 
+  - JSON RPC API (useful for console and Mist integration)
+  - Morden testnet and private network
+  - `ethminer` miner integration (allows *mantis* to mine blocks with `ethminer`)
```

Comment:
As there are several ethminer repos, maybe adding the link might be cool. I've used this one https://github.com/Genoil/cpp-ethereum